geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/SortedSetOptions.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+package org.apache.geode.redis.internal.executor.sortedset;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+import org.apache.geode.DataSerializer;
+import org.apache.geode.internal.serialization.DataSerializableFixedID;
+import org.apache.geode.internal.serialization.DeserializationContext;
+import org.apache.geode.internal.serialization.KnownVersion;
+import org.apache.geode.internal.serialization.SerializationContext;
+
+/**
+ * Class representing different options that can be used with Redis string SET command.
+ */
+public class SortedSetOptions implements DataSerializableFixedID {
```

Comment:
Hmmm...

Reply:
Hale and I created a BaseSetOptions class that SetOptions and SortedSetOptions can extend.

Reply:
Fantastic, thanks for doing that! I think the naming might be a little confusing though, since `SetOptions` is actually related to the `RedisString` class, not `RedisSet`, which makes the name `SortedSetOptions` a bit inconsistent or potentially misleading. Maybe they could be `StringSetOptions` and `SortedSetSetOptions`? That last one is a bit awkward though, so I dunno...

Reply:
Better, I think, to make the SortedSetOption to be ZAddOptions.

Reply:
Ah, yeah, that's way better